tools/pyteomics/mztab2tsv.xml

```diff
@@ -0,0 +1,98 @@
+<tool id="mztab2tsv" name="pyteomics: convert mztab to tabular" version="@TOOL_VERSION@" profile="20.01" license="TODO_license_in_spdx_format">
```

Comment:
maybe the name just "pyteomics" and the rest as description?

license is missing ... its the license of the python script

Reply:
preferences anyone .. ? The license question is always a bit annoying to me .. I would like https://en.wikipedia.org/wiki/WTFPL :)